app/models/deletion.rb

```diff
@@ -0,0 +1,38 @@
+class Deletion < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Should we log the username instead of the user_id. In case we delete the user, we still want to keep the Deletion entry.